### ISSUE TITLE
Accessibility issues - elements must have sufficient color contrast

### ISSUE THREAD

**Issue by flyingfisch** [JasonFritsche/motivationalCats#69]
[Axe](https://www.deque.com/axe/) finds 4 instances of insufficient color contrast on this website.

![image](https://user-images.githubusercontent.com/1307001/95108416-43e34500-0709-11eb-9918-82d5f3dd489b.png)

**Comment by JasonFritsche** [comment]
@flyingfisch would you like me to assign this issue to you?

**Comment by flyingfisch** [comment]
I was just adding it for visibility, I'm fine with taking it or we can see if someone else wants to pick it up. Up to you

**Comment by JasonFritsche** [comment]
Ok cool, let's give it a couple of days and see.

**Comment by AxSmasher44** [comment]
Hello, I would like to help.

**Comment by AxSmasher44** [comment]
hey, could you please review my PR. :)
